tests/integration/popup/popup_core.js

```diff
@@ -8,49 +8,66 @@
 		originalAnimationComplete = $.fn.animationComplete,
 		animationCompleteCallCount = 0,
 		opensAndCloses = function( eventNs, popupId, linkSelector, contentSelector ) {
-			var $popup = $( document.getElementById( popupId ) ),
+			var popup = $( document.getElementById( popupId ) ),
```

Comment:
Yeah, it was to avoid having to ```$.mobile.path.hashToSelector()```, but I guess we control what gets passed in, so I'll change it.